### PR TITLE
Document combining of qualifiers with examples

### PR DESCRIPTION
**Pull request by nicojs**
Add 2 examples, one with a logical `and` and one with a logical `or`

### CODE REVIEW

**Comment by nicojs** [comment]
Thanks for the awesome npms.io! We're integrating it right into our cli!

**Comment by satazor** [comment]
Thanks!

Yes it will, just need to run the docs npm script!

File: lib/routes/search/query.js [hunk 1]
```diff
@@ -17,7 +17,14 @@ module.exports = (router, esClient) => {
      *
      * @apiDescription
      *
-     * Besides normal text, `q` supports qualifiers to express filters and other modifiers:
+     * Besides normal text, `q` supports qualifiers to express filters and other modifiers. Combining qualifiers is possible with a comma (`,`) for a logical or and a plus (`+`) for a logical and.
```
Comment (satazor) [inline lib/routes/search/query.js]: Can we get a new line here?

File: lib/routes/search/query.js [hunk 2]
```diff
@@ -17,7 +17,14 @@ module.exports = (router, esClient) => {
...
+     * Examples:
```
Comment (satazor) [inline lib/routes/search/query.js]: Can we get a new line here?

File: lib/routes/search/query.js [hunk 3]
```diff
@@ -17,7 +17,14 @@ module.exports = (router, esClient) => {
      *
      * @apiDescription
      *
-     * Besides normal text, `q` supports qualifiers to express filters and other modifiers:
+     * Besides normal text, `q` supports qualifiers to express filters and other modifiers. Combining qualifiers is possible with a comma (`,`) for a logical or and a plus (`+`) for a logical and.
+     * Examples: 
+     *  - `https://api.npms.io/v2/search?q=keywords:stryker-plugin+keywords:jasmine`
```
Comment (satazor) [inline lib/routes/search/query.js]: I think these examples should be at the bottom, near the other @apiExample ones

**Comment by satazor** [comment]
@nicojs I have requested some changes but you have not fulfil them. Will close this MR but feel free to open it again.

**Pull request closed without merging**